### DISCUSSION TITLE
A problem in running "How to install software"

### DISCUSSION THREAD

**Discussion by Polan0317** [biostars/biostar-handbook-issues#184] [General]
Hi!
I have a problem in running vcf.
After I key in make vcf, it shows as below:

mkdir -p reads
fastq-dump -F -X 10000 --split-files -O reads SRR1553425
Failed to call external services.
make: *** [Makefile:110: reads/SRR1553425_1.fastq] Error 64

How do I resolve this problem?
Many thanks!

**Comment by ialbert** [comment]
I found some answers here:

https://github.com/ncbi/sra-tools/issues/302

it seems that it happens when you don't have internet access
